web/themes/custom/votegov/src/sass/components/hero.scss

```diff
@@ -57,11 +58,27 @@
   @include at-media-max('tablet-lg') {
     @include grid-container;
   }
+
+  .usa-button {
+    font-family: $theme-font-type-sans;
+    font-size: 16x;
+    width: auto;
+
+    @include at-media-max('mobile') {
+      width: 193px;
+    }
+  }
+
+  p {
+    @include u-margin-top(2);
+    @include u-margin-bottom(3);
+    font-family: $theme-font-type-sans;
+    font-size: 19.2px;
+  }
```

Comment:
Create a new wrapper element around the field body print statement in the template with a class of "vote-hero__text usa-intro". Make sure it's wrapped in a conditional so it won't print if the body field is empty. Then replace this style with ` > * + *` and apply `@include u-margin-top(3);`